pkgs/racket-doc/scribblings/reference/eval-model.scrbl

```diff
@@ -179,8 +179,8 @@ definitions in response to evaluating forms such as @racket[define].
 Each evaluation step, then, transforms the current set of definitions and
 program into a new set of definitions and program. Before a
 @racket[define] can be moved into the set of definitions, its
-second expression must be reduced to a @tech{value}.
-(The first expression is used without evaluation.)
+expression (i.e., its right-hand side) must be reduced to a @tech{value}.
+(The left-hand side is used without evaluation.)
```

Comment:
This is a definite improvement, but I still have a minor quibble with the final parenthetical. It is a category error to talk about “evaluation” of the left-hand side, even in the negative, since the left hand side is not an expression. It would be like saying the `define` in `(define x 3)` is used without evaluation—it’s simply a part of the syntax of the program.

That said, it seems like you felt this needed clarifying, so I imagine there’s a reason you added the extra sentence. Could you describe what you were trying to disambiguate? There might be a better way of doing that.

Reply:
I'm trying to make it clear that the name or procedure being defined is not evaluated in the context of `define`. We could, after all, require `(define 'foo ...)`. 

A minor point is that I find "left-hand side" and "right-hand side" to be odd, since the `define` form does not look like an operator.

Reply:
@lexi-lambda Why can I reply to the first of your comments right above?

Anyway, what do you suggest instead of "indicates the position of". I think "takes the place of" suggests that the redex will never be recomposed into the expression.

Reply:
> We could, after all, require `(define 'foo ...)`

No, we couldn’t do that, since Racket is a statically-bound, lexically-scoped language. Predictable expansion of macros necessitates knowing the binding structure of a program statically, but if we allowed something like `(define (string->symbol (read-line)) ....)`, then we wouldn’t be able to know the binding structure until the program is run.

The “left-hand side” and “right-hand side” terminology is, admittedly, a little weird in the context of Racket. If you want, you could use other phrases instead, but at that point, I think you end up with the original documentation, which just describes “its expression” (unambiguous since there is only one, as described in the documentation for `define`).

Reply:
How about "where [] holds the position of the redex"?

Reply:
[I'm confused about why this conversation is duplicated above.]